app/views/users/new.html.erb

```diff
@@ -0,0 +1,37 @@
+<h1>new</h1>
+<!--
+<form action="/users" method="post" accept-charset="UTF-8">
+  <input type="hidden" name="authenticity_token" value="<%= form_authenticity_token %>">
+  <label for="username">Username:</label>
+  <input type="text" id="username" name="username"><br><br>
+  <label for="email">Email:</label>
+  <input type="text" id="email" name="email"><br><br>
+  <label for="password">Password:</label>
+  <input type="password" id="password" name="password"><br><br>
+  <input type="submit" value="Submit">
+</form>
+-->
+
+<!--
+ <%= form_tag("/users", method: "post") do %>
```

Comment:
- [ ] *[Required]* The action here should also be `'users_path'`, not `'/users'`